bind/motor-mobile/object_builder.go

```diff
@@ -129,3 +130,54 @@ func RemoveObjectField(name, fieldName string) error {
 	}
 	return nil
 }
+
+func BuildObject(name string) ([]byte, error) {
+	if instance == nil {
+		return nil, errWalletNotExists
+	}
+
+	builder, ok := objectBuilders[name]
+	if !ok {
+		return nil, fmt.Errorf("no object builder with name '%s'", name)
+	}
+
+	res, err := builder.Build()
+	if err != nil {
+		return nil, err
+	}
+
+	// Using JSON marshalling here for arbitrary object types
+	return json.Marshal(res)
+}
+
+func UploadObject(name string) ([]byte, error) {
+	if instance == nil {
+		return nil, errWalletNotExists
+	}
+
+	builder, ok := objectBuilders[name]
+	if !ok {
+		return nil, fmt.Errorf("no object builder with name '%s'", name)
```

Comment:
nit: perhaps we should move this a global var for object errors.

Reply:
I'm not sure what you mean. A global object for storing errors that occur in motor?

Reply:
I mena more make it a global var in the bind layer. but we dont have this setup really yet. Ill check to see if there is an issue. resolving this comment.